### PR TITLE
Feat: add back static IP setting UI

### PR DESCRIPTION
**Pull request by andy89923**
## Description

- [x] Add back Static IP UI
- [x] Get SMF Config Data via OAM service
- [x] Verify Static IP button

## Note
- Depend on https://github.com/free5gc/smf/pull/99

### CODE REVIEW

**Comment by andy89923** [comment]
@yccodr Please review this PR~ Thanks! 😀

**Comment by ianchen0119** [comment]
Hi @andy89923 

Did you test the functionality with 5GC?

**Comment by andy89923** [comment]
> Hi @andy89923
> 
> Did you test the functionality with 5GC?

Yes, it could assign the IP setting in the webconsole.

However, I am working on a better coding style in frontend. So mark it as draft for now.

**Review by yccodr — APPROVED** [review]

File: backend/WebUI/api_verify.go [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+	}
+
+	// Check not used by other UE
+	smDataColl := "subscriptionData.provisionedData.smData"
+	filter := bson.M{
+		"singleNssai": snssai,
+		"ueId":        bson.D{{Key: "$ne", Value: checkData.Supi}}, // not this UE
+	}
+	smDataDataInterface, mongo_err := mongoapi.RestfulAPIGetMany(smDataColl, filter)
+	if mongo_err != nil {
```
Comment (ianchen0119) [inline backend/WebUI/api_verify.go:177]: if mongo_err != nil && mongo_err != mongo.ErrNoDocuments {

**Comment by ianchen0119** [comment]
@andy89923  

The IP verifier responds the status 500 once the verification failed (mongo nil document).
I think we should use status 204 instead of status 500. When frontend receives the status 204, it should allow user to set the static ip (cause this ip hasn't been used so far).

**Comment by andy89923** [comment]
> @andy89923
> 
> The IP verifier responds the status 500 once the verification failed (mongo nil document). I think we should use status 204 instead of status 500. When frontend receives the status 204, it should allow user to set the static ip (cause this ip hasn't been used so far).

I think the mongoapi would handle the job. 
If there is no document related, the mongoapi will return an empty slice and nil. 

Here is part of code in mongoapi:
```go
func RestfulAPIGetMany(
  collName string, filter bson.M, argOpt ...interface{},
) ([]map[string]interface{}, error) {
        collection := Client.Database(dbName).Collection(collName) 

        var cur *mongo.Cursor
	var err error
	collation := getCollation(argOpt...)
	if collation != nil {
		opts := new(options.FindOptions)
		opts.SetCollation(collation)
		cur, err = collection.Find(ctx, filter, opts)
	} else {
		cur, err = collection.Find(ctx, filter)
	}

        ...
}
```

**Review by ianchen0119 — APPROVED** [review]

**Pull request merged**